src/address.js

```diff
@@ -1,14 +1,16 @@
 var Buffer = require('safe-buffer').Buffer
 var bech32 = require('bech32')
-var bs58check = require('bs58check')
+var bbs58check = require('./bs58check')
```

Comment:
There is no good reason for overriding this dependency. Just create a new module `bs58checkBlake256` for example. Don't touch any unrelated code to your change.